<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRotCtl</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="Azimuth_lay">
          <item>
           <widget class="QwtCompass" name="az_compass">
            <property name="lowerBound">
             <double>0.000000000000000</double>
            </property>
            <property name="upperBound">
             <double>360.000000000000000</double>
            </property>
            <property name="scaleMaxMajor">
             <number>16</number>
            </property>
            <property name="scaleMaxMinor">
             <number>360</number>
            </property>
            <property name="scaleStepSize">
             <double>22.500000000000000</double>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="lineWidth">
             <number>4</number>
            </property>
            <property name="origin">
             <double>270.000000000000000</double>
            </property>
            <property name="minScaleArc">
             <double>0.000000000000000</double>
            </property>
            <property name="maxScaleArc">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="az_number">
            <property name="maximum">
             <number>450</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="Elevation_lay">
          <item>
           <widget class="QwtDial" name="el_dial">
            <property name="upperBound">
             <double>180.000000000000000</double>
            </property>
            <property name="totalSteps">
             <UInt>180</UInt>
            </property>
            <property name="lineWidth">
             <number>4</number>
            </property>
            <property name="origin">
             <double>180.000000000000000</double>
            </property>
            <property name="minScaleArc">
             <double>0.000000000000000</double>
            </property>
            <property name="maxScaleArc">
             <double>180.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="el_number">
            <property name="maximum">
             <number>180</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="go_button">
        <property name="text">
         <string>Go</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionConnect_to_rotator"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect_to_rotator"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normalon>:/resources/exit.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/settings.svg</normaloff>:/resources/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionConnect_to_rotator">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/connected.svg</normaloff>
     <normalon>:/resources/connected.svg</normalon>:/resources/connected.svg</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to rotator</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/disconnected.svg</normaloff>:/resources/disconnected.svg</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtCompass</class>
   <extends>QwtDial</extends>
   <header>qwt_compass.h</header>
  </customwidget>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
